integrations/vercel.mdx

```diff
@@ -1,4 +1,110 @@
 ---
-title: "Vercel"
-url: "https://github.com/onkernel/vercel-template"
----
\ No newline at end of file
+title: "Vercel Marketplace"
+description: "Integrate Kernel through the Vercel Marketplace"
+---
+
+## Integrate Kernel through the Vercel Marketplace
+
+The Vercel Marketplace allows Vercel users to install and configure third-party services, like Kernel, directly from the Vercel dashboard. Kernel offers an [official integration](https://vercel.com/marketplace/kernel) that Vercel users can install to integrate Kernel into their Vercel projects.
+
+### Installing the Kernel Integration
+
+{/* <Tip>
+  Deploy an example Next.js application and install the Kernel integration with our [template](https://github.com/onkernel/vercel-template).
+</Tip> */}
+
+1. Navigate to the [Kernel integration](https://vercel.com/marketplace/kernel) in the Vercel Marketplace.
+2. Click **Install** to add Kernel to your Vercel team.
+3. Select your pricing plan.
+4. Provide a name for your Kernel installation. (this is not used for billing or anything else)
```

Comment:
Knitpicking here..maybe this for grammatical accuracy? 

Provide a name for your Kernel installation (this is not used for billing or anything else).